app/__init__.py

```diff
@@ -7,19 +7,47 @@
 
 """
 
-from flask import Flask
+from flask import Flask, make_response, jsonify
 from instance.config import app_config
 from .api.v1 import v1 as version_one
 from .api.v2 import v2 as version_two
 from app.db_config import connect, create_tables
 
+
 def create_app(config_name="development"):
+    """
+    Function used to create the Flask application.
+    Takes a string defining the configuration name as a parameter to use when
+    setting application
+    configuration variables such as debugging and database selection.
+
+    Returns
+    --------
+    flask application object with settings configured
+    """
     app = Flask(__name__)
     app.url_map.strict_slashes = False
     app.config.from_object(app_config[config_name])
     app.config['BUNDLE_ERRORS'] = True
-    conn=connect(app.config['DB_NAME'])
+    conn = connect(app.config['DB_NAME'])
     create_tables(conn)
     app.register_blueprint(version_one)
     app.register_blueprint(version_two)
+
+    @app.errorhandler(403)
+    def forbidden(error):
+        return make_response(jsonify({
+            "status_code": 403,
+            "msg": ("You do not have adequate priviledges"
```

Comment:
make this message more direct like: "You do not have permission to do this"

Reply:
Noted, will adjust to make more specific changes, thank you.